packages/@aws-cdk/aws-codebuild/test/codebuild.test.ts

```diff
@@ -1613,7 +1600,7 @@ describe('ARM image', () => {
             computeType: codebuild.ComputeType.MEDIUM,
           },
         });
-      }).toThrow(/ARM images only support ComputeType 'BUILD_GENERAL1_LARGE' - 'BUILD_GENERAL1_MEDIUM' was given/);
+      }).toThrow(/ARM images only support ComputeTypes 'BUILD_GENERAL1_SMALL' and 'BUILD_GENERAL1_LARGE' - 'BUILD_GENERAL1_MEDIUM' was given/);
     });
 
```

Comment:
Can you please add a test here: `'can be used with ComputeType LARGE'`?

Reply:
This is covered by the first test case already which checks for the default "LARGE" ComputeType

Reply:
Nope! It does not. If it did, it would have caught the bug I talk about in https://github.com/aws/aws-cdk/pull/16635#discussion_r715867110.

Can you add it please?